.kitchen.yml

```diff
@@ -1,18 +1,21 @@
 ---
 driver:
-  name: vagrant
+  name: dokken
```

Comment:
while docker is faster, vagrant is more universal and easier for supporting windows and linux.